index.html

```diff
@@ -0,0 +1,135 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Landing Page</title>
+    <link rel="stylesheet" href="css/style.css" />
+    <style>
+      body {
+        display: flex;
+        flex-direction: column;
+        background-color: #eee;
+      }
+
+      @media screen and (min-width: 200px) {
+        body {
+          background-color: #eee;
+        }
+      }
+    </style>
+  </head>
+
+  <body>
+    <header id="header">
+      <div class="logo">
+        <img
+          src="https://cdn.freecodecamp.org/testable-projects-fcc/images/product-landing-page-logo.png"
+          id="header-img"
+        />
+      </div>
+      <nav id="nav-bar">
+        <ul>
+          <li><a class="nav-link" href="#features">Features</a></li>
+          <li><a class="nav-link" href="#how-it-works">How It Works</a></li>
+          <li><a class="nav-link" href="#pricing">Pricing</a></li>
+        </ul>
+      </nav>
+    </header>
+
+    <div></div>
+
+    <div id="hero">
+      <h2>Check Out Here</h2>
+      <form id="form" action="https://www.freecodecamp.com/email-submit">
+        <input
+          type="email"
+          id="email"
+          name="email"
+          placeholder="email"
+          required
+        />
+        <input type="submit" value="submit" id="submit" />
+      </form>
+    </div>
```

Comment:
For better code readability, user experience and greater accessibility consider using semantic tags. In this case, you can use `section` tag instead of div.